hal/src/nRF52840/hal_platform_nrf52840_config.h

```diff
@@ -98,4 +98,6 @@
 
 #define HAL_PLATFORM_NEWLIB (1)
 
-#define HAL_PLATFORM_EXPORT_STDLIB_RT_DYNALIB (1)
\ No newline at end of file
+#define HAL_PLATFORM_EXPORT_STDLIB_RT_DYNALIB (1)
```

Comment:
The only item added with manual conflict resolution is the [XIP one](https://github.com/particle-iot/device-os/pull/2325/files#diff-cd15e2d616a41d7609cfbcd7081989117ef3592b0a8d1d86f0a814e864252116R103). This one still shows up probably because of new line.